cohesive-vns3-free-try-now/azuredeploy.json

```diff
@@ -1,629 +1,628 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
-    "contentVersion": "1.0.0.0",
-    "parameters": {
-        "adminUsername": {
-            "type": "string",
-            "metadata": {
-                "description": "User Name for VM's (required but not used for VNS3)"
-            }    
-        },
-        "adminPassword": {
-            "type": "securestring",
-            "metadata": {
-                "description": "Admin Password for VM's"
-            }
-        },
-        "numberOfInstances": {
-            "type": "int",
-            "minValue": 1,
-            "maxValue": 5,
-            "metadata": {
-              "description": "Hosts to deploy"
-            }
-        },
-        "location": {
-            "type": "string",
-            "defaultValue": "[resourceGroup().location]",
-            "metadata": {
-                "description": "Deployment location"
-            }
-        }, 
-        "virtualMachineOs": {
-            "type": "string",
-            "defaultValue": "ubuntu",
-            "allowedValues": [
-                "ubuntu",
-                "windows"
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "User Name for VM's (required but not used for VNS3)"
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "defaultValue": "GEN-PASSWORD",
```

Comment:
the "GEN" tokens only work in the parameter file, remove the defaultValue and move this back to the param file